instrument/devices/damm.py

```diff
@@ -0,0 +1,43 @@
+"""
+Damm in station 8-ID-D
```

Comment:
```suggestion
Damm (dynamic aperture) in station 8-ID-D
```

I'm guessing this is a dynamic aperture.  Should confirm.  It helps me to translate DAMM from jargon.